api-catalog-ui/frontend/cypress/integration/e2e/dashboard/dashboard.test.js

```diff
@@ -1,4 +1,12 @@
-/* eslint-disable no-undef */
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
```

Comment:
probably not because it wasn't failing and these files did not have the header, but now the linter for cypress is properly configured and it complains about these missing license headers.